@xen-orchestra/fs/src/smb-mount.js

```diff
@@ -1,14 +1,28 @@
 import { parse } from 'xo-remote-parser'
 
 import MountHandler from './_mount'
+import normalizePath from './_normalizePath'
 
 export default class SmbMountHandler extends MountHandler {
   constructor(remote, opts) {
-    const { domain, host, password, username } = parse(remote.url)
+    const {
+      domain = 'WORKGROUP',
+      host,
+      options,
+      password,
+      path,
+      username,
+    } = parse(remote.url)
     super(remote, opts, {
       type: 'cifs',
-      device: `//${host}`,
-      options: `user=${username},password=${password},domain=${domain}`,
+      device: '//' + host + normalizePath(path),
```

Comment:
Does it work?
Is it really possible to add a path here?

Reply:
If yes, then great! :slightly_smiling_face:

Reply:
To be honest, I took over the code from your commit "proposal".

Reply:
You need to check the man or at least test it works (spoiler it does).